api-docs/docs/browser-tracker/browser-tracker.api.md

```diff
@@ -215,6 +215,12 @@ export interface EnableAnonymousTrackingConfiguration {
     stateStorageStrategy?: StateStorageStrategy;
 }
 
+// Warning: (ae-forgotten-export) The symbol "GetBatch" needs to be exported by the entry point index.module.d.ts
```

Comment:
Shouldn't we follow this recommendation and export the `GetBatch` and `PostBatch` types?

Reply:
😅 yes, that seems like a good idea